### PR TITLE
Add GraalVM JDK 17 and 20 versions

### PR DESCRIPTION
**Pull request by alfeilex**
This PR adds the new releases of GraalVM JDK versions to `ide-urls`

Related issue: #1325

### CODE REVIEW

File: ide/src/main/java/com/devonfw/tools/ide/url/updater/graalvm/GraalVmUrlUpdater.java [hunk 1]
```diff
@@ -21,13 +22,20 @@ protected String getVersionPrefixToRemove() {
...
   @Override
   protected void addVersion(UrlVersion urlVersion) {
-
-    String baseUrl = "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${version}/graalvm-ce-java11-";
-    doAddVersion(urlVersion, baseUrl + "windows-amd64-${version}.zip", WINDOWS, X64);
-    doAddVersion(urlVersion, baseUrl + "linux-amd64-${version}.tar.gz", LINUX, X64);
-    doAddVersion(urlVersion, baseUrl + "darwin-amd64-${version}.tar.gz", MAC, X64);
-    doAddVersion(urlVersion, baseUrl + "darwin-aarch64-${version}.tar.gz", MAC, ARM64);
-
+    // if (urlVersion.getName().startWith("jdk")
```
Comment (alfeilex) [inline ide/src/main/java/com/devonfw/tools/ide/url/updater/graalvm/GraalVmUrlUpdater.java:25]: What is the best condition to use here? Is not it easier to use the `urlVersion.getName()` method?

Comment (hohwille) [reply ide/src/main/java/com/devonfw/tools/ide/url/updater/graalvm/GraalVmUrlUpdater.java:25]: Yes that would be better. You could also try something like `urlVersion.getStart().getLettersString()` to get the potential `jdk` prefix but the out-commented code is maybe the one most easy to understand by different developers.

Comment (alfeilex) [reply ide/src/main/java/com/devonfw/tools/ide/url/updater/graalvm/GraalVmUrlUpdater.java]: Done

**Comment by alfeilex** [comment]
I chose `community` and `oracle` to distinguish the editions

**Review by hohwille — APPROVED** [review]
@alfeilex thanks for your rework, now functionally correct 👍 
Ready for merge...
However, an optimization for the future could consider:
* the two non-abstract updaters should filter out the versions they are unable to process and that will only lead to download errors.
* Instead of reading the tagged versions twice from github we could follow a similar approach like we did for intellij and handle both editions in a single updater.

**Pull request merged**